index.html

```diff
@@ -8,24 +8,58 @@
     <link rel="stylesheet" href="css/style.css" />
     <link rel="preconnect" href="https://fonts.googleapis.com" />
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Poppins:wght@300&display=swap"
-      rel="stylesheet"
-    />
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300&display=swap" rel="stylesheet"/>
+    <script src="https://cdn.jsdelivr.net/npm/luxon@1.27.0/build/global/luxon.min.js" defer></script>
   </head>
 
   <body>
-    <h1>Awesome Books</h1>
-    <section id="booksDisplay"><ul></ul></section>
-    <form action="#" id="bookForm">
-      <br>
+    <nav>
+      <ul class="navbar">
+        <li>
+          <a onclick="displaySection(document.getElementById('booksDisplay'))"
+            >List</a
+          >
+        </li>
+        <li>
+          <a onclick="displaySection(document.getElementById('addBook'))"
+            >Add New</a
+          >
+        </li>
+        <li>
+          <a onclick="displaySection(document.getElementById('contact'))"
+            >Contact</a
+          >
+        </li>
+      </ul>
+    </nav>
```

Comment:
- [ ] [OPTIONAL] This is a minor one. Your `nav` section is missing the `Awesome Books` Page title to the left. Please work on this. It's required even though I have listed it as optional.